src/assertions.ts

```diff
@@ -366,6 +369,81 @@ export async function runAssertion({
     };
   }
 
+  if (baseType === 'is-sql') {
+    let parsedSQL;
```

Comment:
nit, prefer google naming conventions https://google.github.io/styleguide/jsguide.html#naming
```suggestion
    let parsedSql;
```

Reply:
got it